requirements-min.txt

```diff
@@ -1,6 +1,6 @@
 # minimum versions of package dependencies for installing PyNWB
 h5py==2.10  # support for selection of datasets with list of indices added in 2.10
-hdmf==3.1.1
+hdmf @ git+https://github.com/catalystneuro/hdmf.git@add_measurement_vector_data
```

Comment:
Unfortunately this will not clone the repo with the submodule, so the CI breaks

Reply:
Maybe it needs a `--recurse-submodule` option on it? I've never worked with this many nested git submodules before, lol